HW1/P2/P2b.py

```diff
@@ -1,3 +1,20 @@
 from P2 import *
 
 # Your code here
+
+# part 2b
+# make pixels
+i = sc.parallelize(range(2000))
+j = sc.parallelize(range(2000))
+
+pxls = i.cartesian(j)
+
+
+# upt through mandelbrot func
+result = pxls.map(lambda x: (x , mandelbrot(x[1]/500.0 - 2,x[0]/500.0 - 2))).partitionBy(100)
```

Comment:
I actually also initially tried to randomize using partitionBy, and it seems we got the same results - better, but still a bit skewed.